src/runner.js

```diff
@@ -4,13 +4,25 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
 
 class Runner {
-  constructor() {
+  /**
+   * Creates an instance of Runner.
+   * @param {object} [options] Options for the runner
+   * @param {string} options.language Options for the language of the formater
+   * @memberof Runner
+   */
+  constructor(options) {
     this.yarnNodes = {};
     this.variables = new DefaultVariableStorage();
     this.functions = {};
     this.visited = {}; // Which nodes have been visited
+    this.options = {
+      language: 'en', // Handle messageFormat language change
+      ...options || {} // rewrite and insert if options contain data
```

Comment:
this is still being deconstructed

Reply:
Why not using spread operator (just curious) ?

Reply:
its not supported by older versions of nodejs as you already noticed, but also older browsers. Believe it or not some smartphones still run on those